### PR TITLE
FORMS-15164 AEM Forms: To localize sample JSON files in GitHub.com

### PR DESCRIPTION
**Pull request by alp86773**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

**Comment by alp86773** [comment]
@rismehta 
Could you please review and merge

**Comment by rismehta** [comment]
Thanks, I have created another PR with the same changes, https://github.com/adobe/aem-core-forms-components/pull/1362

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/af-clientlibs/core-forms-components-runtime-all/resources/i18n/ko-kr.json [hunk 1]
```diff
@@ -1,40 +1,88 @@
...
-    "month": "month",
-    "months": "months",
-    "year": "year",
-    "years": "years",
-    "more": "more",
-    "less": "less"
+  "FileCloseAccessText": "Enter 키를 눌러 파일 삭제",
+  "FileSizeGreater": "파일 ${0}이(가) 예상 크기 ${1}MB를 초과합니다.",
+  "FileNameInvalid": "파일 이름이 (.)으로 시작하거나, \\ / : * ? \" < > | ; % $를 포함하거나, nul, prn, con, lpt 또는 com과 같이 예약된 키워드인 파일은 첨부하지 마십시오.",
+  "FileMimeTypeInvalid": "File(s) ${0} are unsupported file types",
```
Comment (rismehta) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/af-clientlibs/core-forms-components-runtime-all/resources/i18n/ko-kr.json:5]: @alp86773 This is still not translated, can you check this ?

Comment (alp86773) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/af-clientlibs/core-forms-components-runtime-all/resources/i18n/ko-kr.json:5]: @rismehta 
Please use the old one "파일 ${0}은(는) 지원되지 않는 파일 유형입니다."

**Pull request closed without merging**